jest.config.js

```diff
@@ -34,6 +34,9 @@ module.exports = {
   testTimeout: 30000,
   resolver: `<rootDir>/public/test/jest-resolver.js`,
   setupFilesAfterEnv: ['./public/test/setupTests.ts'],
+  testEnvironmentOptions: {
```

Comment:
this is the recommendation from the author of msw: see https://github.com/mswjs/msw/issues/1786#issuecomment-1782559851

little nervous about this change as it affects all the tests 😬